packages/server-core/src/social/invite/invite.hooks.ts

```diff
@@ -45,14 +56,92 @@ import {
   inviteResolver
 } from './invite.resolvers'
 
+async function addSearch(context: HookContext<InviteService>) {
+  if (!context.params.query || !context.params.query.search) {
+    return
+  }
+
+  const search = context.params.query.search
+
+  context.params.query = {
+    ...context.params.query,
+    $or: [
+      {
+        inviteType: {
+          $like: '%' + search + '%'
+        }
+      },
+      {
+        passcode: {
+          $like: '%' + search + '%'
+        }
+      }
+    ]
+  }
+}
+
+function checkQueryType(type: InviteQuery['type']) {
```

Comment:
Can we change this service to use `action` instead of `type` in query? We already have `isAction(NAME)` hook which we can use. Also this will make things sync with other services?

Reply:
implemented!